cypress/integration/ui/sign.test.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+/// <reference types="@testing-library/cypress"/>
+
+const validXDR =
```

Comment:
Please put this in a fixture file which you can inmport, and call it for instance `paymentXdr`